### ISSUE TITLE
[Bug] slow media downloads android

### ISSUE THREAD

**Issue by nzarbayezid** [ankidroid/Anki-Android#10655]
###### Reproduction Steps

1. On desktop creating some deck
2. sync it to ankiweb
3. attempt to sync on Ankidroid

###### Expected Result
downloads all media files in a couple of minutes cuz I have a 1000mbps internet speed

###### Actual Result
Downloading only up to 30kb

###### Debug info
AnkiDroid Version = 2.15.6

Android Version = 12

Manufacturer = Xiaomi

Model = M2102J20SG

Hardware = qcom

Webview User Agent = Mozilla/5.0 (Linux; Android 12; M2102J20SG Build/SKQ1.211006.001; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/99.0.4844.88 Mobile Safari/537.36

ACRA UUID = dcd0b687-e3f8-49ef-b4e2-e66f7d310a18

Scheduler = std2

Crash Reports Enabled = true

DatabaseV2 Enabled = true

###### Research
*Enter an [x] character to confirm the points below:*

- [ x] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid
- [x ] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [x ] I have searched for similar existing issues here and on the user forum
- [x ] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by david-allison** [comment]
Fixable with manual work if you don't mind copying a folder, we're getting this fix into 2.16:

https://github.com/ankidroid/Anki-Android/issues/8925#issuecomment-976337268

Closing as duplicate of #8925, thanks for the report!

**Comment by nzarbayezid** [comment]
thanks for the fast response.
 but, the next update or what?

**Comment by mikehardy** [comment]
We are literally blocked by Google from releasing an update until it is fixed, so yes, it will be in the next official update.
It'll be in our alpha versions here https://github.com/ankidroid/Anki-Android/releases/ you can subscribe to notifications on #5304 to be notified when we've completed the work that will close the issue, then the next alpha will have it, followed by an official release some time after

**Comment by nzarbayezid** [comment]
thanks, I already did download the new alpha but it wasn't fixed that's why I asked, thanks for your explanation I'm gonna wait till it will be fixed
